source/MRViewer/ImGuiHelpers.cpp

```diff
@@ -724,4 +762,34 @@ void SetTooltipIfHovered( const std::string& text, float scaling )
     ImGui::EndTooltip();
 }
 
+void Separator( float scaling, const std::string& text )
+{
+    if ( ImGui::GetStyle().ItemSpacing.y < 16.0f * scaling )
+    {
+        ImGui::SetCursorPosY( ImGui::GetCursorPosY() + 16.0f * scaling - ImGui::GetStyle().ItemSpacing.y );
+    }
```

Comment:
what is 16 ?

Reply:
because we need such spacing before and after separator by design